nupic/research/frameworks/pytorch/imagenet/mixins/knowledge_distillation.py

```diff
@@ -133,6 +201,66 @@ def get_execution_order(cls):
         return eo
 
 
+class KnowledgeDistillationCL(KnowledgeDistillation):
```

Comment:
Does "CL" stand for "combined loss"? When I first saw this, my first guess was that it meant `CompositeLoss` and was indicating that it needs to be used with `CompositeLoss`.

Reply:
It does. I initially used CompositeLoss mixin, because I needed all memory possible and I was worried of leaving all the soft targets in memory (which is the only way to do it with the existing approach of transform data and target + separate loss function). However, upon realizing the targets have a very small memory footprint, even when one they are soft targets and not just indices, I switched to the current approach.

Reply:
The name composite loss is due to the loss function being composed of of several partial loss functions in this approach. Might not be a good term after all, I just ran out of creativity =)